firstvoices/backend/models/async_results.py

```diff
@@ -53,3 +57,28 @@ class Meta:
 
     def __str__(self):
         return self.site.title + " - " + str(self.latest_recalculation_date)
+
+
+class BulkVisibilityJob(BaseJob):
+    """Model to store job results for bulk visibility changes."""
+
+    class Meta:
+        verbose_name = _("bulk visibility job")
+        verbose_name_plural = _("bulk visibility jobs")
+        rules_permissions = {
+            "view": predicates.is_superadmin,
+            "add": predicates.is_superadmin,
+            "change": predicates.is_superadmin,
+            "delete": predicates.is_superadmin,
+        }
+
+    from_visibility = models.IntegerField(
+        choices=Visibility.choices, default=Visibility.TEAM, db_index=True
+    )
+
+    to_visibility = models.IntegerField(
+        choices=Visibility.choices, default=Visibility.TEAM, db_index=True
```

Comment:
Any particular reason for adding db_index on these? I don't think we'll ever be querying or ordering by these values

Reply:
Oh true, I had left them in as db_index is set to true for our other visibility values, but I agree that is this use case the index is probably not going to be used. I will remove it!